### PR TITLE
Introduce Simple Project Fixtures

### PR DESCRIPTION
**Pull request by thehabes**
GitHub Pages is deploying this PR branch as https://static.t-pen.org.  The following links go to the fixtures being introduced.  I collected them all together under a `simple-project` directory as a contained example.  Do we want to include a small README to go along with it?

- https://static.t-pen.org/fixtures/simple-project/project.json
- https://static.t-pen.org/fixtures/simple-project/manifest.json

- https://static.t-pen.org/fixtures/simple-project/layer.json

- https://static.t-pen.org/fixtures/simple-project/page.json

- https://static.t-pen.org/fixtures/simple-project/line.json
- https://static.t-pen.org/fixtures/simple-project/group.json
- https://static.t-pen.org/fixtures/simple-project/user/bryan.json
- https://static.t-pen.org/fixtures/simple-project/user/patrick.json
- https://static.t-pen.org/fixtures/simple-project/user/victor.json

### CODE REVIEW

**Comment by thehabes** [comment]
Testing

File: fixtures/simple-project/user/bryan.json [hunk 1]
```diff
@@ -0,0 +1,13 @@
+{
+   "_id":"661d43bc490d045b8ba04d38",
+   "type":"User",
+   "name":"Bryan H.",
```
Comment (cubap) [inline fixtures/simple-project/user/bryan.json:3]: We talk about `displayName` in other places. Is this essentially what that is? Should we be using a `profile.nickname` as a rule?

File: fixtures/simple-project/user/bryan.json [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+   "email":"bryan@rerum.io",
+   "agent":"https://store.rerum.io/v1/id/661d43bc490d045b8ba04d38",
```
Comment (cubap) [inline fixtures/simple-project/user/bryan.json]: strike `agent` per conversation

File: fixtures/simple-project/user/bryan.json [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+   "profile":{
+      "nickname":"Habes",
+      "github_username":"thehabes",
+      "playstation_username":"thehabes",
+      "favorite_number":13
+   }
```
Comment (cubap) [inline fixtures/simple-project/user/bryan.json:8]: maybe the `sub` from Auth0 is good to record here as a backup?

File: fixtures/simple-project/user/bryan.json (hunk 1 above)
Comment (cubap) [reply fixtures/simple-project/user/bryan.json:3]: Maybe?  I think I took this from an example of what a User looks like in the `users` table, and I think that is probably meant to be something like the 
`displayName`
`handle`
`_tpen3_handle` or `_tpen3_username` in essence

Comment (cubap) [reply fixtures/simple-project/user/bryan.json:3]: Let's commit in the fixtures to something TPEN will always use and recommend for a display name within the application.

If it is always in `profile` then the public can see it, if not, we need to explicitly attach it for non-auth'd requests.

Comment (cubap) [reply fixtures/simple-project/user/bryan.json:3]: Upvote your fav:

Comment (cubap) [reply fixtures/simple-project/user/bryan.json:3]: `name`

Comment (cubap) [reply fixtures/simple-project/user/bryan.json:3]: `nickname`

Comment (cubap) [reply fixtures/simple-project/user/bryan.json:3]: `displayName`

Comment (cubap) [reply fixtures/simple-project/user/bryan.json:3]: `profile.` + above option

File: fixtures/simple-project/user/bryan.json (hunk 3 above)
Comment (thehabes) [reply fixtures/simple-project/user/bryan.json:8]: sure, it is harmless to have around and gives us a direct `getById` call to Auth0

File: fixtures/simple-project/user/bryan.json (hunk 2 above)
Comment (cubap) [reply fixtures/simple-project/user/bryan.json]: The winner is `profile.displayName`

**Pull request merged**